entries.py

```diff
@@ -10,6 +10,22 @@
 	pronunciation="", 
 	seealso="")
 
+API = GlossEntry("API", 
+	acronym_full="Application Programmer Interfaces", 
+	definition="Application Programmer Interface technologies serve as software-based intermediaries to exchange data.", 
+	furtherreading="", 
+	institute="", 
+	pronunciation="", 
+	seealso="")
+
+AWS = GlossEntry("AWS", 
+	acronym_full="Amazon Web Services", 
+	definition="A provider of cloud services available on-demand.", 
```

Comment:
Is there any information we should include here to contrast AWS against GCP?

Reply:
Hmm, not sure how to be succinct with that. They offer different instance types and may have different cloud costs?